integration/hurl/tests_ok/deprecated.hurl

```diff
@@ -1,4 +1,3 @@
-# HTTP/* is deprecated, use HTTP instead
 GET http://localhost:8000/hello
-HTTP/* 200
+HTTP 200
```

Comment:
We can remove this integration test file `integration/hurl/tests_ok/deprecated.hurl` as there is no deprecation to test now